site/mkdocs.yml

```diff
@@ -35,6 +35,7 @@ theme:
     - navigation.tabs
     - navigation.tabs.sticky
     - navigation.path
+    - navigation.sections
```

Comment:
what is this for? 

sounds like its for rendering groups
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections

Reply:
Yes, such that the links to other language implementations are always visible at the top level.